tox.ini

```diff
@@ -13,9 +13,11 @@ whitelist_externals =
     bash
     echo
 
-[testenv:flake8]
-deps = flake8
-commands = flake8 galaxy_api tests
+[testenv:linters]
+commands =
+    pipenv install --dev
```

Comment:
Full set of dependencies and dev dependencies is unnecessary to run `flake8` and `yamllint` tools.

Reply:
@cutwater change amended. Thanks for comment.